test/Microsoft.Framework.PackageManager.FunctionalTests/project.json

```diff
@@ -3,6 +3,7 @@
         "Microsoft.Framework.FunctionalTestUtils": "1.0.0-*",
         "Microsoft.Framework.Runtime.Common": "1.0.0-*",
         "Microsoft.Framework.PackageManager": "1.0.0-*",
+        "Microsoft.Framework.PackageManager.Tests": "1.0.0-*",
```

Comment:
We used to do it the other way; Unit tests depended on functional tests. They share some Utilities. But now that the unit tests have dnxcore50 as a target framework and the functional tests "don't", moving the utilities to the unit tests makes it all work.